### PR TITLE
[WIP] Feat: Printer

### PR DESCRIPTION
**Pull request by mathvaillant**
# ⚠️ READ ⚠️

### The only Java file I create was the  CustomModules.java to expose our Printer to React Native.

570b5304-513a-444e-8a2a-a75069807443

### CODE REVIEW

**Comment by joel-gouveia** [comment]
[Printer Setup](https://trello.com/c/9O3uHkPZ/10-printer-setup)

**Comment by joaoqueiroz95** [comment]
nice

File: src/config/i18n/translations/en.json [hunk 1]
```diff
@@ -31,5 +31,31 @@
...
+    },
+    "thank-message": "Thank you!",
+    "vat-percentage": "{{value}}%",
+    "tse-info-divider": "-----------------TSE Information----------------",
+    "ticket": {
+      "quantity-and-price": "{{quantity}} x at {{price}}",
+      "include-vat": "incl. {{value}}% VAT"
+    },
+    "tse": {
+      "deviceNumber": "Device Serial Number: {{value}}",
```
Comment (joaoqueiroz95) [inline src/config/i18n/translations/en.json:50]: What is tse? I am not sure it is the best name to use

File: src/utils/__tests__/currency.test.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  return {
+    ...requireActual,
+    NativeLocaleIdentifier: "de-DE",
+  };
+});
+
+describe("utils/currency", () => {
+  it("Should format price with currency for EUR(DE)", () => {
+    // @ts-ignore
+    const expected = normalizeWhitespace("10,00 €");
```
Comment (joaoqueiroz95) [inline src/utils/__tests__/currency.test.ts:21]: Why ts-ignore?

File: src/config/i18n/translations/en.json (hunk 1 above)
Comment (mathvaillant) [reply src/config/i18n/translations/en.json:50]: This is the property that comes from the API. Look at the mirror api at:

```
/mocks/resources/receipt/receipt.ts
```

File: src/utils/__tests__/currency.test.ts (hunk 2 above)
Comment (mathvaillant) [reply src/utils/__tests__/currency.test.ts]: Forgot to remove that one, gone now. Danke!

**Review by joaoqueiroz95 — APPROVED** [review]

**Review by joel-gouveia — APPROVED** [review]

**Pull request merged**